Sprint-3/1-key-implement/3-get-card-value.js

```diff
@@ -8,7 +8,22 @@
 // write one test at a time, and make it pass, build your solution up methodically
 // just make one change at a time -- don't rush -- programmers are deep and careful thinkers
 function getCardValue(card) {
```

Comment:
Your function is written quite well. There are a few tests I ran that didn't return the result I would expect. Can you figure out what is missing in your implementation?
```
const tenOfDiamonds = getCardValue("10♦");
assertEquals(tenOfDiamonds, 10,)

const twentytwo = getCardValue("22");
assertEquals(twentytwo, 'Invalid card rank.');

const nothing = getCardValue("");
assertEquals(nothing, 'Invalid card rank.');
```

Reply:
Can you review the above☝️failing tests and implement something within your function to ensure that they don't fail!

Reply:
@delmorallopez I don't see all of the suggested tests here in your PR.

How are you tracking whether you're completed comments? I recommend clicking "Resolve conversation" when you've completed something, so that any unresolved conversations still show up and you know you need to complete them.